### PR TITLE
Feature/orcid

### PR DESCRIPTION
**Pull request by iimpulse**
> Clicking on username gives you a profile and allows you to add an orcid so that users get credit for creating annotations when we put this in the release ecosystem

### CODE REVIEW

**Comment by ielis** [comment]
Hi @iimpulse , the widget seems to be working on my end. I was able to login and add my ORCID. After re-logging, the ORCID was there.
I found one unexpected behavior though. To submit ORCID, the user must click on his user ID on the top right part of the screen. However, if the user clicks, say, 5x on the ID, 5 dialog windows will pop up. I suppose only one of such windows should appear.

**Review by ielis — APPROVED** [review]
Looks good to me!

File: src/main/poet/src/app/shared/services/curation/curation.service.ts [hunk 1]
```diff
@@ -401,9 +400,9 @@ export class CurationService {
   getUserContributions() {
     return this.httpClient.get(environment.POET_API_STATISTICS_CONTRIBUTION_URL).pipe(
       map((contributions) => {
-        return [{"value": contributions["treatment"], "name": "Medical Action Ontology"},
-          {"value": contributions["phenotype"], "name": "Human Phenotype Ontology"},
-          {"value": contributions["phenopackets"], "name": "PhenoPackets"}];
+        return [{value: contributions["treatment"], name: "Medical Action Ontology"},
```
Comment (anna-lamoureux) [inline src/main/poet/src/app/shared/services/curation/curation.service.ts:403]: Did you want to convert these double quotes to single quotes as well?

**Review by anna-lamoureux — APPROVED** [review]
Just a minor suggestion but it's not going to break anything if it's not addressed

File: src/main/poet/src/app/shared/services/curation/curation.service.ts (hunk 1 above)
Comment (iimpulse) [reply src/main/poet/src/app/shared/services/curation/curation.service.ts:403]: I CANT DO THEM ALL REEEE

**Pull request merged**